src/main/kotlin/blackjack/Player.kt

```diff
@@ -1,9 +1,31 @@
 package blackjack
 
-class Player(name: String) : User(name) {
+class Player(name: String, val price: Int) : User(name) {
+
+    init {
+        require(price > 0) { "베팅 금액은 반드시 0보다 커야합니다." }
+    }
+
     override fun getFirstDeal(): List<Card> {
         return listOf(cardDeck.cards[0], cardDeck.cards[1])
     }
+
+    override fun getEvaluate(users: Users): Int {
+        val dealer = users.users.filterIsInstance(Dealer::class.java).first()
```

Comment:
아래와 같이 조금 더 코틀린스러운 코드로 개선해보세요. 😄 
```suggestion
        val dealer = users.users.filterIsInstance<Dealer>().first()
```

Reply:
Dealer 또는 Player들을 찾아서 반환하는 기능은 Users 클래스의 역할에 더 어울리는 것 같아요.